packages/wrangler/src/config/environment.ts

```diff
@@ -648,3 +650,10 @@ export type ConfigModuleRuleType =
 	| "CompiledWasm"
 	| "Text"
 	| "Data";
+
+export type TailConsumer = {
```

Comment:
This should have `binding` property too (or is `service` intended to be exposed on the env?)

Reply:
nope, actually nothing gets exposed to the env for a tail producer when you assign a tail consumer - it's a passive listener, just like wrangler tail / any other trace installs